### ISSUE TITLE
CI scripts broken by #22073

### ISSUE THREAD

**Issue by justinsb** [kubernetes/kubernetes#22287]
Getting this error... looks like we are quoting `ci/release`:

```
+ gsutil cat 'gs://kubernetes-release/'\''ci/latest'\''.txt'
CommandException: No URLs matched gs://kubernetes-release/'ci/latest'.txt
```

Working on a hotfix...

**Comment by justinsb** [comment]
Linking to #22073

**Comment by justinsb** [comment]
Fix is #22288
